README.md

```diff
@@ -491,13 +497,56 @@ manager.getCredentials(new BaseCallback<Credentials, CredentialsManagerException
 ```
 
 
-5. **Clear credentials**
+5. **Clear credentials:**
 When you want to log the user out:
 
 ```java
 manager.clearCredentials();
 ```
 
+
+### Encryption enforced (Min API 21)
```

Comment:
Is the language used typically `min API X` or `API X+` ?

Reply:
i.e. `Retrofit requires at minimum Java 7 or Android 2.3.`